pdl-compiler/src/backends/rust_no_allocation/packet_parser.rs

```diff
@@ -44,12 +44,18 @@ pub fn generate_packet(
         quote! {}
     };
 
+    // computed_offsets/values are HashMaps, so we need to sort to get deterministic results
+    let mut computed_offsets_sorted = curr_schema.computed_offsets.iter().collect::<Vec<_>>();
+    computed_offsets_sorted.sort();
+    let mut computed_values_sorted = curr_schema.computed_values.iter().collect::<Vec<_>>();
+    computed_values_sorted.sort();
+
     let computed_getters = empty()
         .chain(
-            curr_schema.computed_offsets.iter().map(|(decl, defn)| decl.declare_fn(defn.compute())),
```

Comment:
Would you consider changing the type of computed_offsets to BTreeMap instead to force ordered iteration?

Reply:
Done